### ISSUE TITLE
Al levantar container ml_classifier se rompe - AttributeError: 'NoneType' object has no attribute 'drivername'

### ISSUE THREAD

**Issue by rodowi** [mxabierto/sara#91]
Unexpected error: <type 'exceptions.AttributeError'>
Traceback (most recent call last):
  File "ml_classifier.py", line 217, in <module>
    db.create_all()
  File "/usr/local/lib/python2.7/dist-packages/flask_sqlalchemy/__init__.py", line 895, in create_all
    self._execute_for_all_tables(app, bind, 'create_all')
  File "/usr/local/lib/python2.7/dist-packages/flask_sqlalchemy/__init__.py", line 887, in _execute_for_all_tables
    op(bind=self.get_engine(app, bind), **extra)
  File "/usr/local/lib/python2.7/dist-packages/flask_sqlalchemy/__init__.py", line 833, in get_engine
    return connector.get_engine()
  File "/usr/local/lib/python2.7/dist-packages/flask_sqlalchemy/__init__.py", line 496, in get_engine
    self._sa.apply_driver_hacks(self._app, info, options)
  File "/usr/local/lib/python2.7/dist-packages/flask_sqlalchemy/__init__.py", line 775, in apply_driver_hacks
    if info.drivername.startswith('mysql'):
AttributeError: 'NoneType' object has no attribute 'drivername'

**Comment by rodowi** [comment]
creo que es porque le falta SQLALCHEMY_URL al docker-compose

**Comment by rodowi** [comment]
Error

```
  File "/root/sara/ml_classifier.py", line 102, in get_auth_token
    token = g.user.generate_auth_token()
  File "/root/sara/ml_classifier.py", line 58, in generate_auth_token
    return s.dumps({'id': self.id})
  File "/usr/local/lib/python2.7/dist-packages/itsdangerous.py", line 742, in dumps
    return signer.sign(self.dump_payload(header, obj))
  File "/usr/local/lib/python2.7/dist-packages/itsdangerous.py", line 353, in sign
    return value + want_bytes(self.sep) + self.get_signature(value)
  File "/usr/local/lib/python2.7/dist-packages/itsdangerous.py", line 348, in get_signature
    sig = self.algorithm.get_signature(key, value)
  File "/usr/local/lib/python2.7/dist-packages/itsdangerous.py", line 268, in get_signature
    mac = hmac.new(key, msg=value, digestmod=self.digest_method)
  File "/usr/lib/python2.7/hmac.py", line 136, in new
    return HMAC(key, msg, digestmod)
  File "/usr/lib/python2.7/hmac.py", line 71, in __init__
    if len(key) > blocksize:
TypeError: object of type 'NoneType' has no len()
```

Se soluciona agregando SECRET_KEY a docker-compose o al docker run

**Comment by rodowi** [comment]
Fixed by #93
